cmake/crypto.cmake

```diff
@@ -24,7 +24,7 @@ if("sgx" IN_LIST COMPILE_TARGETS)
   )
   target_compile_definitions(
     evercrypt.enclave PRIVATE INSIDE_ENCLAVE KRML_HOST_PRINTF=oe_printf
-                              KRML_HOST_EXIT=oe_abort
+                              KRML_HOST_EXIT=oe_abort LINUX_NO_EXPLICIT_BZERO
```

Comment:
What does that flag do?

Reply:
This tells the build that there's no [explicit_bzero](https://www.freebsd.org/cgi/man.cgi?query=explicit_bzero&sektion=3). I don't know what exactly that's used for though.

Reply:
Huh. There doesn't seem to be any calls to `bzero()` in the evercrypt code. musl as provided in openenclave does expose both `bzero()` and `explicit_bzero()` though, so should we have this flag at all?

Reply:
I see this:

    root@45ddd2ed032b:/CCF/3rdparty/hacl-star/evercrypt# grep bzero *
    Lib_Memzero0.c:    explicit_bzero(dst, len_);

If it's properly supported by musl we may want to update the macro-checks in `Lib_Memzero0.c` (the build explicitly asks for pull requests...)